docs/examples/POWGEN_data_reduction.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "936a13e1-c9ea-4351-b29d-7dae9ad1e073",
+   "metadata": {},
+   "source": [
+    "# POWGEN data reduction\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook gives a concise overview of how to use the ESSDiffraction package with Sciline.\n",
+    "It uses a simple reduction workflow for the SNS [POWGEN](https://sns.gov/powgen) experiment.\n",
+    "\n",
+    "We begin with relevant imports:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b160f394-de0b-4b73-9aa5-c13ac3a1d707",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "import scippneutron as scn\n",
+    "import sciline\n",
+    "\n",
+    "import ess.diffraction\n",
+    "from ess.diffraction import powder\n",
+    "from ess.diffraction.external import powgen\n",
+    "from ess.diffraction.types import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6270f661-099f-4f8a-9380-76050f73f251",
+   "metadata": {},
+   "source": [
+    "## Define reduction parameters\n",
+    "\n",
+    "We define a dictionary containing the reduction parameters.\n",
+    "The keys are types defined in [essdiffraction.types](../generated/modules/ess.diffraction.types.rst)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "67e8d584-016d-422e-add9-a61e5b1fe8bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params = {\n",
+    "    # Input data\n",
+    "    Filename[SampleRun]: 'PG3_4844_event.zip',\n",
+    "    Filename[VanadiumRun]: 'PG3_4866_event.zip',\n",
+    "    CalibrationFilename: 'PG3_FERNS_d4832_2011_08_24.zip',\n",
+    "\n",
+    "    # Crop data to this range in time-of-flight\n",
+    "    ValidTofRange: sc.array(dims=['tof'], values=[0.0, 16666.67], unit='us'),\n",
+    "\n",
+    "    # Edges for binning in d-spacing\n",
+    "    DspacingBins: sc.linspace('dspacing', 0.0, 2.3434, 200, unit='angstrom'),\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3d9b9bd-6923-4124-954f-e92672845be2",
+   "metadata": {},
+   "source": [
+    "## Create pipeline using Sciline\n",
+    "\n",
+    "We use the basic providers available in `essdiffraction` as well as the specialised `powder` and `powgen` providers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "35412429-c039-4591-b8b8-49b11a128b7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "providers = [\n",
+    "    *ess.diffraction.providers,\n",
+    "    *powder.providers,\n",
```

Comment:
This looks a bit confusing, `powder` is a submodule of `ess.diffraction`? What is the split? Do you want to explain to the reader?

Reply:
I thought the goal was to also have single crystal code in this project. So I kept the old module structure from `ess`.

Reply:
I don't know. I thought that was the reason you suggested a common package.

Reply:
My point is: You have `ess.diffraction.providers`, implying there is something common?

Reply:
Yes. Many providers are regular functions that were already defined in the common diffraction part of `ess`. I don't know whether they can be used by SXD. Esp. as providers because the domain types are probably all different.

So what do you suggest? Move everything into `powder`? Or remove `powder` and move everything into `diffraction`?

Reply:
I think you can leave it, but maybe add a comment?

Reply:
Opened an issue to track this and make us re-examine when we add sxd code.